modules/microshift-configuring-hosts-for-mirror.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * microshift/running_applications/microshift-deploy-with-mirror-registry.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-configuring-hosts-for-mirror_{context}"]
+= Configuring hosts for mirror registry access
+
+To configure a {product-title} host to use a mirror registry, you must give the host access and create a configuration file with the host name and port through which the registry can be reached.
+
+.Prerequisites
+* Your mirror host has access to the internet.
+* You configured a mirror registry to use in your restricted network and can access the certificate and credentials that you configured.
```

Comment:
I'm not sure what that part after "and" means.

Reply:
Whoo, that was end-of-day sentence. I think I meant,

* Your mirror host has access to the internet.
* The mirror host can access the mirror registry.
* You configured a mirror registry to use in your restricted network.
* You downloaded the pull secret and modified it to include authentication to your mirror repository.

I must have been thinking of the authentication needed to pull the images.

Reply:
Yes, let's replace the original text with the 4 bullet points you mentioned in the comment. I think it's great.